src/network/networkd-network-gperf.gperf

```diff
@@ -194,6 +194,7 @@ IPv6AcceptRA.UseDNS,                         config_parse_bool,
 IPv6AcceptRA.UseDomains,                     config_parse_dhcp_use_domains,                            0,                             offsetof(Network, ipv6_accept_ra_use_domains)
 IPv6AcceptRA.RouteTable,                     config_parse_section_route_table,                         0,                             0
 IPv6AcceptRA.BlackList,                      config_parse_ndisc_black_listed_prefix,                   0,                             0
+DHCPServer.WaitForUplink,                    config_parse_bool,                                        0,                             offsetof(Network, dhcp_server_wait_for_uplink)
```

Comment:
Is this necessary to be configurable?

Reply:
If we enable this by default and no uplink is ever established, the DHCP server will never be started. That seems unintuitive and something that would definitely annoy me as a user (and is technically a breaking change as well). Instead, we could add a timeout after which we start the DHCP server regardless of whether we have an uplink or not but hardcoding the timeout would be annoying for use cases where it takes a long time for an uplink to become available so then we'd want to make the timeout configurable which would also require adding an option.

Reply:
I see. The timeout is not a good choice.